src/components/TextSize.js

```diff
@@ -5,9 +5,12 @@ import { articleTextSize } from 'utils'
 export const TextSize = ({ close }) => {
   const i18n = useI18n()
 
-  const [textsize, setTextSize] = useArticleTextSize()
-  const { onKeyArrowLeft, onKeyArrowRight } = articleTextSize.getSoftkeyEffect(setTextSize)
+  const [fontSizeClass, setFontSizeClass] = useArticleTextSize('TextSize')
+  const { onKeyArrowLeft, onKeyArrowRight } = articleTextSize.getSoftkeyEffect(setFontSizeClass)
   const sliderValue = ['0', '16.6', '33.2', '49', '66.6', '83.2', '94']
+  const parseTextSizeInteger = (fontSizeClass) => {
```

Comment:
hmmm i don't understand this line, can you explain a bit?

Reply:
The purpose of this method is simply to grab number "x" (as integer) of the "font-size-x" class name, which I use to access the appropriate element of the `sliderValue`